package.json

```diff
@@ -29,127 +29,129 @@
     "eslint-plugin-react": "^7.7.0",
     "extract-text-webpack-plugin": "^3.0.2",
-    "file-loader": "^1.1.11",
+    "file-loader": "^2.0.0",
     "filesize": "^3.6.0",
-    "fs-extra": "^6.0.0",
-    "gzip-size": "^4.1.0",
+    "fs-extra": "^7.0.0",
+    "gzip-size": "^5.0.0",
     "html-webpack-plugin": "^3.0.6",
     "http-proxy-middleware": "^0.18.0",
     "jest": "^23.0.0",
     "json-loader": "^0.5.7",
     "object-assign": "^4.1.1",
     "path-exists": "^3.0.0",
-    "postcss-loader": "^2.1.1",
+    "postcss-loader": "^3.0.0",
     "prettier": "^1.11.1",
     "promise": "^8.0.1",
     "prompt": "^1.0.0",
     "react-dev-utils": "^5.0.0",
     "recursive-readdir": "^2.2.2",
     "strip-ansi": "^4.0.0",
-    "style-loader": "^0.21.0",
-    "truffle-solidity-loader": "^0.0.8",
+    "style-loader": "^0.23.0",
+    "truffle-solidity-loader": "^0.1.0",
+    "uglifyjs-webpack-plugin": "^1.3.0",
     "url-loader": "^1.0.1",
     "webpack": "~3",
     "webpack-dev-server": "~2",
     "webpack-manifest-plugin": "~1",
     "whatwg-fetch": "^2.0.3"
+  },
+  "dependencies": {
+    "@decent-bet/contract-slots": "^0.1.0",
+    "@fortawesome/fontawesome-free": "^5.3.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.4",
+    "@fortawesome/free-brands-svg-icons": "^5.0.10",
+    "@fortawesome/free-solid-svg-icons": "^5.0.8",
+    "@fortawesome/react-fontawesome": "^0.1.0",
+    "@material-ui/core": "^3.0.1",
+    "aes256": "^1.0.3",
+    "async": "^2.6.0",
+    "bignumber.js": "^7.0.0",
+    "bip39": "^2.5.0",
+    "bluebird": "^3.5.1",
+    "chai": "^4.1.2",
+    "chart.js": "^2.7.2",
+    "chartnew.js": "^3.9.0",
+    "countdown": "^2.6.0",
+    "crypto-js": "^3.1.9-1",
+    "dotenv": "^6.0.0",
+    "ethereum-units": "0.0.1-b",
+    "ethereumjs-util": "^5.1.5",
+    "ethers": "^3.0.4",
+    "eventing-bus": "^1.3.3",
+    "ipfs-mini": "^1.1.2",
+    "jquery": "^3.3.1",
+    "keythereum": "^1.0.2",
+    "moment": "^2.22.1",
+    "pinnacle-sports": "^1.2.0",
+    "prop-types": "^15.6.1",
+    "react": "^16.2.0",
+    "react-animations": "^1.0.0",
+    "react-copy-to-clipboard": "^5.0.1",
+    "react-dom": "^16.2.0",
+    "react-iframe": "^1.1.0",
+    "react-load-script": "^0.0.6",
+    "react-redux": "^5.0.7",
+    "react-router-dom": "^4.2.2",
+    "react-tap-event-plugin": "^3.0.2",
+    "react-transition-group": "^2.2.1",
+    "redux": "^4.0.0",
+    "redux-actions": "^2.3.0",
+    "redux-logger": "^3.0.6",
+    "redux-promise-middleware": "^5.1.0",
+    "redux-thunk": "^2.2.0",
+    "redux-thunk-actions": "^1.2.1",
+    "request": "^2.86.0",
+    "rxjs": "^6.2.2",
+    "rxjs-compat": "^6.2.2",
+    "scroll-to-element": "^2.0.0",
+    "sketch": "^0.2.0",
+    "solidity-sha3": "^0.4.1",
+    "swarm-js": "^0.1.37",
+    "thor-devkit": "^0.2.3",
+    "thorify": "0.3.6",
+    "truffle-hdwallet-provider": "^0.0.6",
+    "web3": "^1.0.0-beta.35",
+    "wowjs": "^1.1.3"
+  },
+  "scripts": {
+    "start": "node scripts/start.js DEBT_PLATFORM_FRONTEND",
```

Comment:
These'd need to be `DBET_PLATFORM_FRONTEND`

Reply:
fixed